native-image-tests/build.gradle

```diff
@@ -39,29 +39,6 @@ animalsniffer {
   ignoreFailures = true
 }
 
-def graalHome = "/Library/Java/JavaVirtualMachines/graalvm-ce-java11-20.2.0/Contents/Home"
-def graalvmResources = project.file("$buildDir/graalvm/resources")
-def graalvmConfig = project.file("$graalvmResources/META-INF/native-image")
-graalvmConfig.mkdirs()
-
-sourceSets {
-  main.resources.srcDirs += graalvmResources
-
-  main.java.srcDirs += project.file("../okhttp-logging-interceptor/src/test/java")
```

Comment:
Will reenable with each batch of tests

Reply:
https://github.com/square/okhttp/pull/6356
https://github.com/square/okhttp/pull/6355